firebase-ml-modeldownloader/src/main/java/com/google/firebase/ml/modeldownloader/FirebaseModelDownloader.java

```diff
@@ -422,11 +432,10 @@ private CustomModel getLocalModelDetails(@NonNull String modelName) {
   }
 
   /**
-   * Triggers the move to permanent storage of successful model downloads and lists all models
-   * downloaded to device.
+   * Lists all models downloaded to device. Triggers the move to permanent storage of successful
```

Comment:
I wonder, do users care that it trigger the "move to permanent storage etc..."? It's implementation detail, right?